ynr/apps/people/management/commands/people_clean_person_names.py

```diff
@@ -0,0 +1,35 @@
+from candidatebot.helpers import CandidateBot
+from django.core.management.base import BaseCommand
+from people.models import Person
+from popolo.models import OtherName
+
+
+class Command(BaseCommand):
+    help = "Replaces chars in person names that we don't want to be there."
+
+    def handle(self, *args, **options):
+        chars_to_replace = {"\xa0": " ", "\u200e": ""}
+
+        for char, replace in chars_to_replace.items():
+            qs = Person.objects.filter(name__contains=char).only("pk", "name")
+            for person in qs:
+                for other_name in person.other_names.all():
+                    other_name.name = other_name.name.replace(char, replace)
+                    other_name.save()
+                bot = CandidateBot(person_id=person.pk, update=True)
+                bot.edit_field("name", person.name.replace(char, replace))
+                bot.save(
+                    f"Replacing {repr(char)} with {repr(replace)} in person name"
+                )
```

Comment:
It looks like there's quite different handling of primary vs other names. If we're editing the primary name, we get a nice edit attributed to Candidate Bot with a reason for the edit. If we're editing an 'other' name, it looks like we just silently do it?